app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Admin::CategoriesController < ApplicationController
+  def index
+    @categories = Category.all
+  end
+
+  def show
+    @category = Category.find(params[:id])
+  end
+
+  def new
+    @category = Category.new
+  end
+
+  def create
+    @category = Category.new(category_params)
```

Comment:
Please use IF-ELSE statement for saving  new Category, so that we avoid errors in the future :bow: 
Follow your def update function as an example
![image](https://user-images.githubusercontent.com/35232348/103861323-3a96b980-50f8-11eb-96ae-de8c0abc272c.png)